shaders/src/common_math.glsl

```diff
@@ -19,7 +19,6 @@
 #endif
 
 #define saturate(x)        clamp(x, 0.0, 1.0)
-#define atan2(x, y)        atan(y, x)
```

Comment:
Afaict there's no typo here, it matches HLSL's atan2 behavior which is atan2(x, y) == GLSL's atan(y, x)